### PR TITLE
Prevent panic when TotalIPCount is not set

### PR DESCRIPTION
**Pull request by dataclouder**
When the field `TotalIPCount` in NSX-T edge gateway is not set, an attempt to read a data source results in panic.
Fixes issue #1242

### CODE REVIEW

File: .changes/vTDB/1244-bug-fixes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* Fix Issue [#1242](https://github.com/vmware/terraform-provider-vcd/issues/1242): panic when edge gateway IP count returns empty [GH-1244]
```
Comment (dataclouder) [inline .changes/vTDB/1244-bug-fixes.md:1]: Reminder to update this folder name when the release version is set

**Review by lvirbalas — APPROVED** [review]
Neat! Thanks!

**Review by adambarreiro — APPROVED** [review]

**Review by Didainius — APPROVED** [review]

File: .changes/v3.13.0/1244-bug-fixes.md [hunk 2]
```diff
@@ -0,0 +1 @@
+* Fix Issue [#1242](https://github.com/vmware/terraform-provider-vcd/issues/1242): panic when edge gateway IP count returns empty [GH-1244]
```
Comment (adambarreiro) [inline .changes/v3.13.0/1244-bug-fixes.md:1]: If we release this as a bugfix, we can move it already to v3.12.1

Comment (dataclouder) [reply .changes/v3.13.0/1244-bug-fixes.md]: changed

**Pull request merged**